pipenv/core.py

```diff
@@ -103,25 +103,23 @@ def load_dot_env(project, as_dict=False):
             [project_directory, ".env"]
         )
 
-        if os.path.isfile(dotenv_file):
+        if not os.path.isfile(dotenv_file) and project.s.PIPENV_DOTENV_LOCATION:
             click.echo(
-                crayons.normal(fix_utf8("Loading .env environment variables..."), bold=True),
+                "{}: file {}={} does not exist!!\n{}".format(
+                    crayons.red("Warning", bold=True),
+                    crayons.normal("PIPENV_DOTENV_LOCATION", bold=True),
+                    crayons.normal(project.s.PIPENV_DOTENV_LOCATION, bold=True),
+                    crayons.red("Not loading environment variables.", bold=True),
+                ),
```

Comment:
Can `click.secho` be used here instead of crayons?

Crayons is kind of dead.

Reply:
Sounds like a good idea, but also I'd prefer we do that in a follow-up PR that addresses the whole project space.